.gitlab-ci.yml

```diff
@@ -37,6 +37,8 @@ deploy_to_docker_registries:
     IMG_SOURCES: ghcr.io/datadog/dd-trace-js/dd-lib-js-init:$CI_COMMIT_TAG
     IMG_DESTINATIONS: dd-lib-js-init:$CI_COMMIT_TAG
     IMG_SIGNING: "false"
+    RETRY_COUNT: 5
+    RETRY_DELAY: 300
```

Comment:
How was this value picked? Could it be faster? Adding 5 minutes is quite a long time if the first attempt fails.